test/BUILD

```diff
@@ -1,6 +1,6 @@
 package(default_visibility = ["//visibility:public"])
 
-load("//bazel:envoy_build_system.bzl", "envoy_cc_test_library")
+load("//bazel:envoy_build_system.bzl", "envoy_cc_test", "envoy_cc_test_library")
```

Comment:
Is this change needed if there are no other changes in this file? (Mostly trying to understand bazel).